setup.cfg

```diff
@@ -5,7 +5,6 @@ universal = 1
 license_file = LICENSE
 
 [tool:pytest]
-norecursedirs = .* build dist venv test_data piptools/_compat/* piptools/_vendored/*
```

Comment:
I think there's supposed to be deleted only `piptools/_vendored/*`, but not tho hole `norecursedirs`.

On the other hand, [default](https://docs.pytest.org/en/3.4.0/customize.html#confval-norecursedirs) `norecursedirs` (wich is '.*', 'build', 'dist', 'CVS', '_darcs', '{arch}', '*.egg', 'venv') works fine since CI is green. I git blamed this line and found a couple of references: 
1. 0fd5daa64b96fff3906a7a3b0ab5850345e76419 — adds `test_data`
1. 81c7b55830f89bb3705b9d0851936c94993fd155 — adds `piptools/_compat/*` and `piptools/_vendored/`

@suutari @vphilippon do you remember why did we exclude `test_data` and `piptools/_compat/*`? What was the problem with `test_data`? Is this still relevant?

Reply:
@hramezani could you remove only `piptools/_vendored/*` and leave the rest? We'll address the above comment in the following issue/PR.

Reply:
Thanks, I fixed it.

Reply:
@atugushev The reason to ignore `test_data` is stated in the commit message you referenced (0fd5daa):
> setup.cfg: Configure pytest to ignore fake_pypi when collecting tests,
> because collecting the setup.py files in fake_pypi causes problems,
> since pytest imports and runs them when collecting.

This happens when pytest is ran with the `--doctest-modules` flag.

Reply:
@suutari thanks for the clarification. Looks like it's not relevant anymore.

Reply:
OTOH, there is no need to look into the `tets_data` for pytest.